openshift/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+IMAGE=$1
+CA=`oc get secret -n kube-service-catalog -o go-template='{{ range .items }}{{ if eq .type "kubernetes.io/service-account-token" }}{{ index .data "service-ca.crt" }}{{end}}{{"\n"}}{{end}}' | tail -n 1`
+oc process -f openshift/starter-pack.yaml -p IMAGE=$IMAGE -p BROKER_CA_CERT=$CA | oc apply -f -
```

Comment:
github complains unless you have a trailing newline :-/

Reply:
Thanks, addressed